### PR TITLE
Fix/euk subwf remove alignment

### PR DESCRIPTION
**Pull request by KateSakharova**
Removed alignment euk MAG to reads to calculate depth because it can be re-used from initial metabat run.

MultiQC:
- Add FasqQC
- Add MultiQC
- added stats for busco and samtools to MutliQC report

### CODE REVIEW

**Review by Ales-ibt — APPROVED** [review]
It looks Ok to me, It is great you rid of unnecessary bam files. I think most of the commits are to add idxstats to the multiqc report, which is useful to track where the reads goes. I like the multiMap function, it makes the code more clean and tidy.

**Review by chrisAta — APPROVED** [review]
looks good to me thank you!

**Review by Ales-ibt — APPROVED** [review]
Minor changes looks Ok to me :D

File: assets/multiqc_config.yml [hunk 1]
```diff
@@ -15,18 +15,32 @@ export_plots: true
...
+  - busco
 
 ## Module order
-top_modules:
-  - "fastp"
+module_order:
+  - fastqc
+  - fastp
+  - samtools:
+      name: Samtools. Alignment reads on assembly
```
Comment (mberacochea) [inline assets/multiqc_config.yml:28]:
```suggestion
      name: Samtools - Stats from reads aligned to the assembly
```

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -15,18 +15,32 @@ export_plots: true
...
+      path_filters: "*.assembly.idxstats"
+  - samtools:
+      name: Samtools. CONCOCT bins alignment on assembly
```
Comment (mberacochea) [inline assets/multiqc_config.yml:31]:
```suggestion
      name: Samtools - Stats from CONCOCT bins aligned to the assembly
```

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -15,18 +15,32 @@ export_plots: true
...
+      path_filters: "*.concoct.idxstats"
+  - samtools:
+      name: Samtools. MetaBAT2 bins alignment on assembly
```
Comment (mberacochea) [inline assets/multiqc_config.yml:34]:
```suggestion
      name: Samtools - Stats from MetaBAT2 bins aligned to the assembly
```

File: modules/nf-core/multiqc/main.nf [hunk 4]
```diff
@@ -41,7 +43,7 @@ process MULTIQC {
 
     stub:
     """
-    touch multiqc_data
+    mkdir multiqc_data
     touch multiqc_plots
     touch multiqc_report.html
```
[the remark this answers is not in the document]
Comment (mberacochea) [reply modules/nf-core/multiqc/main.nf:48]: Can you please generate the diff file please `nf-core modules patch multiqc`

File: subworkflows/local/euk_mags_generation.nf [hunk 5]
```diff
@@ -188,10 +173,13 @@ workflow EUK_MAGS_GENERATION {
     ch_log = ch_log.mix( DREP_MAGS.out.progress_log )
 
     emit:
-    genomes = GZIP_MAGS.out.compressed.collect()
-    stats = BUSCO_EUKCC_QC.out.eukcc_final_qc
-    coverage = COVERAGE_RECYCLER_EUK.out.mag_coverage.map{ meta, coverage_file -> coverage_file }.collect()
-    taxonomy = BAT_TAXONOMY_WRITER.out.all_bin2classification
-    versions = ch_versions
-    progress_log = ch_log
+    genomes                   = GZIP_MAGS.out.compressed.collect()
```
Comment (mberacochea) [inline subworkflows/local/euk_mags_generation.nf:176]: Do we need to use `.collect()` here?

**Review by mberacochea — CHANGES_REQUESTED** [review]
Thanks @KateSakharova I left some minor comments

File: modules/local/align_bwa/main.nf
Comment (Ales-ibt) [inline modules/local/align_bwa/main.nf:1]: If you have multiple rounds of decontamination, you can use the meta.id of the reference to differentiate them

**Pull request merged**